mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -2364,13 +2364,13 @@ fn collect_orderbook_metrics(ctx: &MmArc, orderbook: &Orderbook) {
     }
 
     let memory_db_size = malloc_size(&orderbook.memory_db);
-    mm_gauge!(ctx.metrics, "orderbook.len", orderbook.order_set.len() as i64);
-    mm_gauge!(ctx.metrics, "orderbook.memory_db", memory_db_size as i64);
+    mm_gauge!(ctx.metrics, "orderbook.len", orderbook.order_set.len() as f64);
+    mm_gauge!(ctx.metrics, "orderbook.memory_db", memory_db_size as f64);
 
     // TODO remove metrics below after testing
     for (pubkey, pubkey_state) in orderbook.pubkeys_state.iter() {
-        mm_gauge!(ctx.metrics, "orders_uuids", pubkey_state.orders_uuids.len() as i64, "pubkey" => pubkey.clone());
-        mm_gauge!(ctx.metrics, "history.commited_changes", history_committed_changes(&pubkey_state.order_pairs_trie_state_history), "pubkey" => pubkey.clone());
+        mm_gauge!(ctx.metrics, "orders_uuids", pubkey_state.orders_uuids.len() as f64, "pubkey" => pubkey.clone());
```

Comment:
It seems that we need to use `mm_counter` instead of `mm_gauge` as we don't expect that `orders_uuids` is float.
Please consider replacing `mm_gauge` with `mm_counter` where it's needed.

Reply:
@borngraced sorry, my mistake.
We can't use `mm_counter` here since it increments corresponding metric by the given value.
I've checked [metrics::Counter](https://docs.rs/metrics/latest/metrics/struct.Counter.html), [metrics::Gauge](https://docs.rs/metrics/latest/metrics/struct.Gauge.html) structures, and it turns out that there is nothing we can do except using `mm_gauge`...
We even can't use [metrics::Counter::absolute](https://docs.rs/metrics/latest/metrics/struct.Counter.html#method.absolute) as it sets the value if and only if it's greater than the previous counter value.